src/screens/GlobalFeed/index.tsx

```diff
@@ -86,6 +86,7 @@ export default function GlobalFeed() {
               onDelete={onDeletePost}
               postDetail={item}
               postIndex={index}
+              isGlobalfeed={true}
```

Comment:
Actually for passing boolean into component you can just pass a props name and put default value for that prop as `false` inside target component like

```tsx
{/* use this pattern */}
<ComponentA thisIsBoolean  />

{/* Instead of */}
<ComponentA thisIsBoolean={true}  />
```
then defined default value for component

```tsx
const ComponentA = ({ thisIsBoolean = false }) => (...)
```